bower.json

```diff
@@ -29,28 +29,25 @@
   "dependencies": {
     "polymer": "^2.0.0",
     "vaadin-themable-mixin": "vaadin/vaadin-themable-mixin#^1.2.0",
-    "vaadin-element-mixin": "vaadin/vaadin-element-mixin#^2.0.0",
-    "vaadin-lumo-styles": "vaadin/vaadin-lumo-styles#^1.3.3",
-    "vaadin-material-styles": "vaadin/vaadin-material-styles#^1.2.0",
+    "vaadin-element-mixin": "vaadin/vaadin-element-mixin#^2.3.0",
+    "vaadin-lumo-styles": "vaadin/vaadin-lumo-styles#^1.6.0",
+    "vaadin-material-styles": "vaadin/vaadin-material-styles#^1.3.2",
     "vaadin-license-checker": "vaadin/license-checker#^2.1.0",
-    "vaadin-text-field": "vaadin/vaadin-text-field#^2.1.5",
-    "vaadin-select": "vaadin/vaadin-select#^2.0.2",
-    "vaadin-list-box": "vaadin/vaadin-list-box#^1.1.0",
-    "vaadin-item": "vaadin/vaadin-item#^2.1.0",
-    "vaadin-grid": "vaadin/vaadin-grid#^5.3.0",
-    "vaadin-checkbox": "vaadin/vaadin-checkbox#^2.2.1"
+    "vaadin-text-field": "vaadin/vaadin-text-field#^2.6.0-alpha4",
+    "vaadin-select": "vaadin/vaadin-select#^2.2.0-alpha1",
+    "vaadin-list-box": "vaadin/vaadin-list-box#^1.3.0-alpha3",
+    "vaadin-item": "vaadin/vaadin-item#^2.2.0-alpha1",
+    "vaadin-grid": "vaadin/vaadin-grid#^5.6.0-alpha3",
+    "vaadin-checkbox": "vaadin/vaadin-checkbox#^2.3.0-alpha1"
   },
   "devDependencies": {
     "iron-component-page": "^3.0.0",
     "iron-test-helpers": "^2.0.0",
     "webcomponentsjs": "^1.0.0",
     "web-component-tester": "^6.1.5",
-    "vaadin-combo-box": "vaadin/vaadin-combo-box#^4.2.1",
-    "vaadin-demo-helpers": "vaadin/vaadin-demo-helpers#^3.0.0",
-    "vaadin-date-picker": "vaadin/vaadin-date-picker#^4.0.0",
+    "vaadin-combo-box": "vaadin/vaadin-combo-box#^5.1.0-alpha1",
+    "vaadin-demo-helpers": "vaadin/vaadin-demo-helpers#^3.1.0-alpha1",
+    "vaadin-date-picker": "vaadin/vaadin-date-picker#^4.1.0-alpha1",
     "vaadin-dialog": "vaadin/vaadin-dialog#^2.2.1"
```

Comment:
Nit: not a blocker. We can wait for the `dialog` release with updated dependencies first

Reply:
This is a dev dependency, so I think we can keep it as it is.